akka-projection-grpc/src/it/resources/db/default-init.sql

```diff
@@ -0,0 +1,96 @@
+-- FIXME duplicated from akka-projection-grpc/ddl-scripts to be available on IT test classpath
```

Comment:
Not sure what do about this, perhaps it is fine?

Reply:
um, changing the schema will be rare

Reply:
Added a comment so we don't forget to update both if we change.